### PR TITLE
Move template field comment_$ into meeting_user

### PR DESCRIPTION
**Pull request by reiterl**
Update example data, fix problems with it, update tests.

### CODE REVIEW

**Review by r-peschke — CHANGES_REQUESTED** [review]

File: global/meta/models.yml [hunk 1]
```diff
@@ -302,12 +302,12 @@ user:
     to: committee/forwarding_user_id
     restriction_mode: E
 
+  meeting_user_ids:
+    type: relation-list
+    to: meeting_user/user_id
+    restriction_mode: B
```
[the remark this answers is not in the document]
Comment (reiterl) [reply global/meta/models.yml:308]: Changed.

File: global/meta/models.yml [hunk 2]
```diff
@@ -452,6 +452,25 @@ user:
...
+meeting_user:
+  id:
+    type: number
+    required: true
+    restriction_mode: A
+  user_id:
+    type: relation
+    to: user/meeting_user_ids
+    required: true
+    restriction_mode: B
```
[the remark this answers is not in the document]
Comment (reiterl) [reply global/meta/models.yml:464]: Changed.

File: global/meta/models.yml [hunk 3]
```diff
@@ -452,6 +452,25 @@ user:
...
+  meeting_id:
+    type: relation
+    to: meeting/meeting_user_ids
+    required: true
+    restriction_mode: B
```
[the remark this answers is not in the document]
Comment (reiterl) [reply global/meta/models.yml:469]: Changed.

File: global/meta/models.yml [hunk 4]
```diff
@@ -452,6 +452,25 @@ user:
     required: True
     restriction_mode: F
 
+meeting_user:
```
[the remark this answers is not in the document]
Comment (reiterl) [reply global/meta/models.yml:455]: Use *A*. Changed.

File: global/meta/models.yml [hunk 5]
```diff
@@ -1181,6 +1200,10 @@ meeting:
     restriction_mode: B
 
   # Users
+  meeting_user_ids:
+    type: relation-list
+    to: meeting_user/meeting_id
```
[the remark this answers is not in the document]
Comment (reiterl) [reply global/meta/models.yml:1205]: Added,

File: global/meta/models.yml [hunk 6]
```diff
@@ -302,12 +302,12 @@ user:
     to: committee/forwarding_user_id
     restriction_mode: E
 
+  meeting_user_ids:
+    type: relation-list
+    to: meeting_user/user_id
```
[the remark this answers is not in the document]
Comment (reiterl) [reply global/meta/models.yml:307]: Added.

File: openslides_backend/action/actions/meeting/delete.py [hunk 7]
```diff
@@ -41,6 +41,12 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
             for user_id in meeting.get("user_ids", [])
         ]
         self.execute_other_action(UserUpdate, action_data)
+        action_data = [
```
[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/action/actions/meeting/delete.py:44]: This is no longer necessary. So removed.

File: tests/system/action/meeting_user/test_create.py [hunk 8]
```diff
@@ -0,0 +1,28 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class MeetingUserCreate(BaseActionTestCase):
+    def test_create(self) -> None:
```
[the remark this answers is not in the document]
Comment (reiterl) [reply tests/system/action/meeting_user/test_create.py:5]: Removed the included tests.

**Review by r-peschke — APPROVED** [review]

**Pull request merged**